scripts/app.js

```diff
@@ -1759,14 +1748,6 @@ var checkAns = function (uAns, rAns, exact) {
     }
     return correct_flag;
 };
-var scrollToStart = function () {
```

Comment:
Should bring this back as manually scrolling up when finishing the tests is tedious.